my-dsl.md

```diff
@@ -2,37 +2,47 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[Zillions of Games](http://www.zillions-of-games.com/) is a board game scripting language.
 
 # Domain
 _Describe the language's domain in five words._
 
+Grid-Based Board Game Rules
 
 # Computational model
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of 
 what happens when a program in your DSL runs._
 
+When a *Zillions of Games*  file is loaded, the game engine creates a board game on the specified grid with the specified rules, and plays the game automatically. 
```

Comment:
Does this mean that the game engine is playing against the user? is it user vs user? or how does it work?